python/ray/serve/api.py

```diff
@@ -260,7 +260,7 @@ def delete_endpoint(self, endpoint: str) -> None:
 
         Does not delete any associated backends.
         """
-        self._get_result(self._controller.delete_endpoint.remote(endpoint))
```

Comment:
@ijrsvt I also removed the future-related code from these calls because it isn't used. I think we may want to move it inside of the `BackendState` object for simplicity.

Reply:
So in essence the async behavior is only really important for updating the number of backends?